salt/pillar/journal.sls

```diff
@@ -11,6 +11,12 @@ journal:
     crm_api_key: ~
     crm_api_site_key: ~
 
+    google_api_client:
+        client_id: some-id
```

Comment:
@lsh-0 is there any need to put these values here or does their inclusion in [builder-configuration](https://github.com/elifesciences/builder-configuration/blob/master/pillar/journal-public.sls#L9-L13) suffice?

Reply:
The values in the formula's `salt/pillar/journal.sls` are used to illustrate which values the formula depends on (documentation) and during development in vagrant. When running masterless (no connection to the master-server) there is a local copy of builder-private available but no copy of builder-configuration (pretty sure).